src/tests/ftest/nvme/enospace.yaml

```diff
@@ -58,3 +58,6 @@ ior:
       transfer_size: 2048  # 2K
     16M:
       nvme_transfer_size: 16777216  # 16M
+
+aggregation:
+  scm_threshold_percent: 99  # percent of scm expected to be reclaimed after aggregation
```

Comment:
This seems to be standard for other tests too. E.g. https://github.com/daos-stack/daos/blob/62468538829718310ada4bb3cbf236b934eba709/src/tests/ftest/erasurecode/restart.yaml#L56-L57